DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -81,18 +111,29 @@ void FillBindingData(
             for (auto& source : sources)
             {
                 assert(source.resource != nullptr);
-                assert(source.resourceDesc != nullptr);
+                
+               if (isSerializedGraph)
 
-                if (!std::holds_alternative<Model::BufferDesc>(source.resourceDesc->value))
                 {
-                    throw std::invalid_argument("DML operators only support buffer bindings");
+                    bindingData.bufferBindings[bufferIndex].Buffer = source.resource;
```

Comment:
In both cases, setting the value of Buffer and offset is same. So can we move it outside of if/else and have it at the common place, just above to line 137?

Reply:
done